src/Controller/Component/AuthorizationComponent.php

```diff
@@ -143,7 +143,7 @@ protected function getIdentity(ServerRequestInterface $request)
         $identityAttribute = $this->getConfig('identityAttribute');
         $identity = $request->getAttribute($identityAttribute);
         if ($identity === null) {
-            throw new MissingIdentityException([$identityAttribute]);
+            return $identity;
```

Comment:
Oh no. This makes unauthorized handlers useless and breaks implementations where missing identifiers should be redirected.

Please revert that change.